geos/entrypoints.go

```diff
@@ -286,3 +286,15 @@ func CoverageUnion(g geom.Geometry) (geom.Geometry, error) {
 func UnaryUnion(g geom.Geometry) (geom.Geometry, error) {
 	return rawgeos.UnaryUnion(g)
 }
+
+// ConcaveHull returns concave hull of input geometry.
+// pctconvex - ratio 0 to 1 (0 - max concaveness, 1 - convex hull)
+// allowHoles - true to allow holes inside of polygons.
+func ConcaveHull(g geom.Geometry, pctconvex float64, allowHoles bool) (geom.Geometry, error) {
+	return rawgeos.ConcaveHull(g, pctconvex, allowHoles)
+}
+
+// ConvexHull returns convex hull of input geometry.
```

Comment:
Can you remove `ConvexHull`? Typically, I haven't been exposing functionality already implemented as part of the `github.com/peterstace/simplefeatures/geom` package (i.e. native Go implementation) unless there is a good reason to.

Reply:
added here as a pair to ConcaveHull. Could be useful for comparing performance of pure go implementation. Anyway, will remove it - ConcaveHull with ratio=1 is ConvexHull